src/App.jsx

```diff
@@ -5,7 +5,7 @@ import "./App.css";
 function App() {
   const [count, setCount] = useState(0);
 
-  return <h1>FROM PULL REQUEST</h1>;
```

Comment:
No Very good (example 👀)

Reply:
@ChrisDietrich405

Reply:
i dot not agree

Reply:
I don't understand, I built the button to the tech lead's specifications

Reply:
I'm not going to fire you